src/ser.rs

```diff
@@ -123,11 +123,7 @@ fn maybe_display_param(writer: &mut impl fmt::Write, key: impl fmt::Display, val
 #[rustfmt::skip]
 impl<'a, T> fmt::Display for Uri<'a, bitcoin::address::NetworkChecked, T> where for<'b> &'b T: SerializeParams {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        if f.alternate() {
```

Comment:
I request these deleted lines from 126 are still included while lowercasing both conditional branches

Reply:
Yes, this is required by me too. It's only about schema, not other parts.